dotcom-rendering/src/model/find-pillar.ts

```diff
@@ -6,10 +6,12 @@ export const findPillar: (
 	tags?: TagType[],
 ) => ArticleTheme | undefined = (name, tags?) => {
 	// Flag paid content for Labs pillar (for styling purposes)
-	const isPaidContent = (tag: TagType) =>
-		tag.type === 'Tone' && tag.id === 'tone/advertisement-features';
+	const hasPaidContent = !!tags?.some(
```

Comment:
doesn't this coerce `tags` into a boolean? can you call `some` on that?

Reply:
[TypeScript thinks it’s fine](https://www.typescriptlang.org/play?#code/MYewdgzgLgBFCGBzCMC8MCy8oAsB0ATvGACYgC2AFAJQwB8MADHgKwwD8MA2gN4wCWJAFwwARFHABTAPTwSAN0kEo-CJPKSwUUTAC+AXRgiArqUkAzfmEkkAULdCQQAG0l5nIRJQCE3hMnY8CApJSko+QT1aVAZI1HixCWtZBSUVNQ0tUWpqIA)